homeassistant/components/sabnzbd/__init__.py

```diff
@@ -23,6 +25,8 @@
 from homeassistant.helpers.event import async_track_time_interval
 from homeassistant.helpers.typing import ConfigType
 
+from ...helpers.device_registry import async_get
```

Comment:
Please use absolute imports for names outside the integration package.

Reply:
Done